.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+name: Release
+
+on:
+  push:
```

Comment:
Let's use Github's release publish event rather than tag push.

Reply:
tags:
      - 'v*'   This will only publish on tags push. This is likely the standard across multiple oss

Reply:
I requested these changes because we were manually creating GitHub releases without pushing the tags. This appears to be a better solution for automating releases. However, I have a few queries regarding this approach:

1. How should we handle actions for any words starting with the character 'v' (e.g., 'vpn' or 'virtual')?
2. How is the CHANGELOG generated when a patch for the previous version is released? (Context: We currently have version 1.2.0, but there is a security issue on version 1.1.0, and we need to release 1.1.1 for a security patch.)
3. How are we planning to maintain the CHANGELOG file in the repository? In my opinion, keeping the CHANGELOG solely on the GitHub releases may not be sufficient.

Reply:
For reference: `axios` has an action that creates a chore PR on every release. https://github.com/axios/axios/blob/v1.x/.github/workflows/pr.yml